extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/ControllerImpl.java

```diff
@@ -551,6 +563,15 @@ private QueryDefinition initializeQueryDefAndState(final Closer closer)
         id(),
         task.getDataSource(),
         context,
+        (failedTask, fault) -> {
+          addToKernelManipulationQueue((kernel) -> {
+            if (isDurableStorageEnabled) {
```

Comment:
I think across this PR and https://github.com/apache/druid/pull/13368, we might need to unify what enabling durable storage, enabling fault tolerance, and enabling durable storage for intermediate steps mean since that might cause confusion in the code as well as to the end user configuring the properties.

Reply:
> IMO, it should default to false at first
Sorry what is it here , `isDurableStorageEnabled` or a new context param `faultTolerant`which is by default false?